### DISCUSSION TITLE
Dropdown configuration?

### DISCUSSION THREAD

**Discussion by EnhancedJax** [fuma-nama/fumadocs#1194] [Q&A]
Hi! Loving the framework so far. I couldn't seem to find documentation on the dropdown that is available on the documentation site, may I know if this is part of the framework or is it just a custom component? Thanks!

![image](https://github.com/user-attachments/assets/43fa9cab-2ae7-495a-9486-115247e21370)

**Comment by fuma-nama** [comment]
https://fumadocs.vercel.app/docs/ui/layouts/docs#sidebar-tabs

**Reply by EnhancedJax** [reply]
Thanks!
